code/modules/modular_computers/file_system/programs/generic/manifest.dm

```diff
@@ -30,16 +30,24 @@
 		data["_PC"] = headerdata
 		. = data
 
+	VUEUI_SET_CHECK_IFNOTSET(data["allow_printing"], !!(computer?.nano_printer), ., data)
 	VUEUI_SET_CHECK(data["manifest"], SSrecords.get_manifest_list(), ., data)
 
 /datum/computer_file/program/manifest/Topic(href, href_list)
 	. = ..()
 
+	if(href_list["action"] == "print" && can_run(usr, 1) && !!(computer?.nano_printer))
+		if(!computer.nano_printer.print_text(SSrecords.get_manifest_text(), text("crew manifest ([])", worldtime2text())))
+			to_chat(usr, SPAN_WARNING(">Hardware error: Printer was unable to print the file. It might be out of paper."))
```

Comment:
Is that `>` a result of you copy-pasting SPAN_WARNINGs and messing up with a span?

Reply:
As with most of the other issues it's caused by just copying the [original code](https://github.com/Aurorastation/Aurora.3/pull/10470/files#diff-f184093de1da6533b4c920bf39996da0a13581d535cc85516bdb0e0819ae50d6L153) and not thinking about it / seeing it. x)